cob_twist_controller/CMakeLists.txt

```diff
@@ -38,12 +38,14 @@ target_link_libraries(constraint_solvers damping_methods inv_calculations)
 add_library(limiters src/limiters/limiter.cpp)
 target_link_libraries(limiters ${catkin_LIBRARIES} ${orocos_kdl_LIBRARIES})
 
+add_library(interface_types src/interface_types/interface_type.cpp)
```

Comment:
I think we at least need to:
`target_link_libraries(interface_types ${catkin_LIBRARIES})`